notebooks/gauss_plot.ipynb

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    X = jnp.linspace(-3, 3, 500)
</pre><p>X should be small.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>